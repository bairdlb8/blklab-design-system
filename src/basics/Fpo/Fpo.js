import React from 'react'
import PropTypes from 'prop-types'

import './Fpo.scss'

const Fpo = ({ className = '', children }) => {
  return <div className={`ds-fpo p-xxl mb-xl text-align-center`}>FPO: {children}</div>
}

Fpo.propTypes = {}

export default Fpo
