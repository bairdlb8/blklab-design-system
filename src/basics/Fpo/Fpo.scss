.ds-fpo {
  font-size: 24px;
  font-weight: bold;
  background-color: #f9f9f9;
  border: dashed 2px #ccc;
}
