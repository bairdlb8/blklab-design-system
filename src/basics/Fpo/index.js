import Fpo from './Fpo'
export default Fpo
