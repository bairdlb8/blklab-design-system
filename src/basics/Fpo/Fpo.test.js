import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'
import Fpo from './Fpo'

expect.extend(toHaveNoViolations)

describe('<Fpo />', () => {

  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Fpo />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Fpo />)

    expect(asFragment(<Fpo />)).toMatchSnapshot()
  })

})
