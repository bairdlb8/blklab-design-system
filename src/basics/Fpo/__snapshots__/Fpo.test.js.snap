// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Fpo /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Fpo
  </div>
</DocumentFragment>
`;
